init.lua

```diff
@@ -615,6 +615,73 @@ function CommDKP_OnEvent(self, event, arg1, ...)
 				CommDKP:LootTable_Set(lootList)
 			end
 		end
+	elseif core.DB.defaults.AutoAwardLoot and event == "TRADE_SHOW" then

```

Comment:
U need to first register the events at  the end of init.lua

events:RegisterEvent("TRADE_SHOW")
events:RegisterEvent("TRADE_ACCEPT_UPDATE")
events:RegisterEvent("TRADE_CLOSED")

Reply:
Whoops, I had copied over the changes from my local installation and missed that one. Thanks!